packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -26,20 +26,11 @@ export function ProgressBar({
   totalSteps,
   size = "base",
 }: ProgressBarProps) {
-  const percentage = ((currentStep - 1) / (totalSteps - 1)) * 100;
-  const widthPercent = Math.min(100, Math.max(0, percentage));
-  const progressBarContainerClassName = classnames(
-    styles.container,
-    sizes[size],
-  );
   const progressBarContentClassName = classnames(styles.content);
 
   return (
-    <div className={progressBarContainerClassName}>
-      <div
-        className={progressBarContentClassName}
-        style={{ width: `${widthPercent}%` }}
-      />
-    </div>
+    <progress max={totalSteps} value={currentStep}>
+      {currentStep} of {totalSteps}
```

Comment:
Works for both steps and loading

ie "248 of 512"

Maybe a fancier way could be adding a prop for type of progress? Steps vs filesize, and units?

But I'm thinking the context of whatever you did before this action would be enough here